css/style.css

```
@@ -16,4 +24,101 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+#nav-bar {position:fixed;
+          top:0; 
+          height: 70px;
+          width:100%;
+          right:40px;
+          background-color: white;
+                   }
 
+    #nav-bar ul {list-style-type:none; 
+                display:flex; 
+                justify-content: flex-end;
+                }     
+  
+    #nav-bar ul a:hover {color:#F15B2A;
+                        }
+
+    #nav-bar img {height:60px; 
+                  width:200px; position: absolute;
+                  left: 10px;
+                  top: 10px;
+                  z-index: -1;
+                 }
+
+    .nav-link {text-decoration:none; 
+              color:darkgray; 
+              font-weight:bolder; 
+              display:block; 
+              padding:8px; 
+             }
+    
+   .nav li:first-child > a {color: rgb(46, 44, 44);}
+
+
+article {margin-top:70px;}
+
+
+   h1 {color: white; font-weight:lighter; font-size: 2em;}
+   h2 {color: white; font-weight:lighter;}
+
+#intro {background-image: url('https://raw.githubusercontent.com/willem-blauenberg/portfolio/master/projects/karma/img/first-background.jpg'); 
+                  height: 700px;
+                  width:100%;
+                  background-size: cover left;
+                  background-position: center;
+                  padding-top:50px;
+                  text-align: center;
+                  background-repeat: no-repeat;
+                        
+                  display: flex;
+                  justify-content: center;
+                  align-items: center;
+                   } 
+
+   #into-content {}
+
+#learnbutton { background-color: #F15B2A;
+        border: none;
+        color: white;
+        padding: 15px 32px;
+        text-align: center;
+        text-decoration: none;
+        display: inline-block;
+       }
+
+#content {height:350px}
+
+#content h1 {color: rgb(46, 44, 44); }
+
+.service-img {width:100px; height:100px; }
+
+.services {width:200px;
+           text-align:center;
+           display:inline-block;
+           float:center;
+           padding:10px;
+           }
+
+.servicename {text-decoration: none; color:rgb(46, 44, 44);}
+
+
+footer {height:220;
```

```suggestion
footer {height:220px;
```
It looks like we're missing units here. Also, we should be using relative units like `rem` as mentioned in my other comment